### ISSUE TITLE
Talkback is not announcing the numbering for the list items and not announcing "Out of List" when the focus moves out of the list items.

### ISSUE THREAD

**Issue by SuryacharanM** [dotnet/maui#6498]
GitHubTags:
#.NET MAUI; #AILimited; #A11ySev2; #A11yMAS; #A11yTCS; #Android; #AndroidPhone; #WCAG1.3.1; #GH_.NETMAUI_Android_Apr2022; #Info and Relationships​; #CEACCApproved; #Closed;

### Description

Talkback is not announcing the numbering for the list items under the search results. Talkback is just announcing "Pizza Dough list items, Tap to activate". It has to announce the numbering for all the list items or it has to announce "Out of List" when the focus moves out of the list items.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. Swipe till "search box" and enter some random data.
3. Swipe till the list items and observe the issue.

https://user-images.githubusercontent.com/86710788/165447868-6aa444d1-dda5-4a78-a746-449e431d82a4.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Relevant log output

```shell
Talkback should announce the numbering for the list items or it has to announce "Out of List" when the focus moves out of the list items.
```

**Comment by PureWeen** [comment]
The list numbering here is not required for this once on Android. Chat with Tanaz if you have questions. 

Next version should have the "Out of List" issue fixed as well.
